File: ts-sdk/src/sdk/models/operations/getserverinfo.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import * as shared from "../shared";


export class GetServerInfoResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  serverInfo?: shared.ServerInfo;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: python-sdk/src/formanceapi/models/operations/getserverinfo.py
import dataclasses
from ..shared import serverinfo as shared_serverinfo
from typing import Optional


@dataclasses.dataclass
class GetServerInfoResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    server_info: Optional[shared_serverinfo.ServerInfo] = dataclasses.field(default=None)
    